rapidtide/workflows/rapidtide_parser.py

```diff
@@ -210,686 +257,933 @@ def _get_parser():
-            help=('Window function to use prior to correlation. '
-                 'Options are hamming (default), hann, '
-                 'blackmanharris, and None. '),
-            default='hamming')
+        "--windowfunc",
+        dest="windowfunc",
+        action="store",
+        type=str,
+        choices=["hamming", "hann", "blackmanharris", "None"],
+        help=(
+            "Window function to use prior to correlation. "
+            "Options are hamming (default), hann, "
+            "blackmanharris, and None."
+        ),
+        default="hamming",
+    )
     wfunc.add_argument(
-            '--nowindow',
-            dest='windowfunc',
-            action='store_const',
-            const='None',
-            help='Disable precorrelation windowing. ',
-            default='hamming')
+        "--nowindow",
+        dest="windowfunc",
+        action="store_const",
+        const="None",
+        help="Disable precorrelation windowing.",
+        default="hamming",
+    )
 
     preproc.add_argument(
-            '--detrendorder',
-             dest='detrendorder',
-             action='store',
-             type=int,
-             metavar='ORDER',
-             help=('Set order of trend removal (0 to disable, default is 1 - linear). '),
-             default=3)
+        "--detrendorder",
+        dest="detrendorder",
+        action="store",
+        type=int,
+        metavar="ORDER",
+        help=(
+            "Set order of trend removal (0 to disable, "
+            "default is 1 - linear)."
+        ),
+        default=3,
+    )
     preproc.add_argument(
-            '--spatialfilt',
-             dest='gausssigma',
-             action='store',
-             type=float,
-             metavar='GAUSSSIGMA',
-             help=('Spatially filter fMRI data prior to analysis '
-                   'using GAUSSSIGMA in mm. '),
-             default=0.0)
+        "--spatialfilt",
+        dest="gausssigma",
+        action="store",
+        type=float,
+        metavar="GAUSSSIGMA",
+        help=(
+            "Spatially filter fMRI data prior to analysis "
+            "using GAUSSSIGMA in mm."
+        ),
+        default=0.0,
+    )
     preproc.add_argument(
-            '--globalmean',
-            dest='useglobalref',
-            action='store_true',
-            help=('Generate a global mean regressor and use that '
-               'as the reference regressor.  If no external regressor is specified, this'
-               'is enatbled by default. '),
-            default=False)
+        "--globalmean",
+        dest="useglobalref",
+        action="store_true",
+        help=(
+            "Generate a global mean regressor and use that as the reference "
+            "regressor.  If no external regressor is specified, this "
+            "is enabled by default."
+        ),
+        default=False,
+    )
 
     globalmethod = preproc.add_mutually_exclusive_group()
```

Comment:
It looks like there's only one `globalmethod` parameter, so I think this group can be removed.

Reply:
Certainly seems that way.  Fine to remove it.

Reply:
Yes, I think you're right.  The group can be removed.